File: content/snippets/python/interview_lcci_16_05_factorial_zeros.py
def trailing_zeroes(n: int) -> int:
    ans = 0
    while n > 0:
        n //= 5
        ans += n
    return ans


def main() -> None:
    print(trailing_zeroes(3))
    print(trailing_zeroes(5))


if __name__ == "__main__":
    main()

File: content/snippets/go/interview_lcci_16_05_factorial_zeros.go
package main

import "fmt"

func trailingZeroes(n int) int {
	ans := 0
	for n > 0 {
		n /= 5
		ans += n
	}
	return ans
}

func main() {
	fmt.Println(trailingZeroes(3))
	fmt.Println(trailingZeroes(5))
}
